en/config/actuators.md

```diff
@@ -1,94 +1,251 @@
+There is an additional field to indicate which servo is used to tilt the motor:
+
+- `Tilted by`: The associated servo used for tilting the motor.
+  The properties of this servo are defined in the [Motor Tilt Servo Geometry](#motor-tilt-servo-geometry).
+
+
+#### Motor Geometry: Standard VTOL
+
+The default standard VTOL ("quadplane") motor geometry is shown below.
+
+![Geometry motor: standard vtol](../../assets/config/actuators/qgc_geometry_standard_vtol_motors.png)
+
+Motors have most of the same configuration fields as for the [multicopter geometry](#motor-geometry-multicopter).
+There is an additional field to indicate the direction in which the motor moves the vehicle (for a standard VTOL, the hover motors are usually set "upwards" and the pusher motor is set to "forwards").
+
+- `Axis`: One of `Upwards`, `Downwards`, `Forwards`, `Backwards`, `Leftwards`, `Rightwards`, `Custom`
+  - If `Custom` is selected, then the UI displays three additional fields for setting the motor orientation.
+
+#### Motor Geometry: Other Vehicles
 
-![Plane Deflections](../../assets/config/actuators/plane_servo_convention.png)
+Other vehicle types will define an appropriate motor geometry for their frame type.
+Once again these motors will generally have the same kinds of properties as shown above.
+
+For example, a fixed-wing vehicle may just have a single pusher moter, while a rover with differential steering will have a motor for throttle and for steering.
+
+#### Motor Position Coordinate System
+
+The coordinate system for motor positions is FRD (in body frame), where the X axis points forward, the Y axis to the right and the Z axis down.
+The origin of the system is the vehicle centre-of-gravity (COG) (**not** the autopilot location).
+
+![Actuators CG reference diagram](../../assets/config/actuators/quadcopter_actuators_cg_reference.png)
+
+#### Bidirectional Motors
+
+Some vehicles may use bidirectional motors (i.e. motor spins in direction 1 for lower output range and in direction 2 for the upper half).
+For example, ground vehicles that want to move forwards and backwards, or VTOL vehicles that have pusher motors that go in either direction.
+
+If bidiectional motors are used, make sure to select the **Reversible** checkbox for those motors (the checkbox is displayed as an "advanced" option).
+
+![Reversible](../../assets/config/actuators/qgc_geometry_reversible.png)
+
+Note that you will need to also ensure that the ESC associated with bidirectional motors is configured appropriately (e.g. 3D mode enabled for DShot ESCs, which can be achieved via [DShot commands](../peripherals/dshot.md#commands)).
+
+
+<!-- For example, the roll scale, pitch scale, slew rate and trim can be set (where appropriate) for a control surface, but the positions of these surfaces to each other and to motors cannot be specified. -->
+
+
+### Control Surfaces Geometry
```

Comment:
This bit below is very hard to read. @junwoo091400 I was thinking of making it more "real" like the way I did for multicopters. So none of this "conceptual" heading stuff - use a real example and then extend it. 

As it is now all the information is there, but I think people will find it hard to map to what they are actually doing in the section.

So actually I'd start with a standard vtol (say) and show its actuators and list all the possible options. I might then link down to explain the specific types.

I will look at this tomorrow.